### PR TITLE
Docs:  Updated the Query inspector page

### PR DESCRIPTION
**Pull request by lwandz13**
Updated the Query inspector page:
- changed the name to Query inspector
- updates to reflect the current UI

### CODE REVIEW

File: docs/sources/explore/explore-inspector.md [hunk 1]
```diff
@@ -1,97 +1,87 @@
...
+## Query inspector JSON tab
 
-- **Show data frame:** Select the result set data you want to view.
-- **Series joined by time:** View the raw data from all of your queries at once, one result set per column. You can click a column heading to sort the data.
+View data results as JSON and as data frame JSON models in the **JSON tab**.
 
-### Download raw query results as CSV
+1. Open the Query inspector and click the **JSON tab**.
+1. Choose one of the following options from the **Select source** dropdown menu:
+   - **Panel data -** Displays a JSON object representing the data retrieved from Explore.
```
Comment (ifrost) [inline docs/sources/explore/explore-inspector.md:58]: Not sure if it matters by we can also specify *what* receives the data

```suggestion
   - **Panel data -** Displays a JSON object representing the data retrieved by the visualization from Explore.
```

File: docs/sources/explore/explore-inspector.md [hunk 2]
```diff
@@ -1,97 +1,87 @@
...
 
-1. In the **Inspector** tab, get raw query results by following the instructions [described in Inspect raw query results](#inspect-raw-query-results).
-1. Refine query settings until you can see the raw data that you want to export.
-1. Click **Download CSV**.
+View, inspect and download raw query results in the **Data tab**.
 
-In order to download a CSV file specifically formatted for Excel, expand **Data options** and then enable the **Download for Excel** toggle before you click the **Download CSV** option.
+1. Open the Query inspector and click the **Data** tab.
+1. Click **Data options** to to view options under **Show data frame**.
+1. Select a data results set from the dropdown menu.
```
Comment (ifrost) [inline docs/sources/explore/explore-inspector.md:67]: The dropdown is shown only when there are multiple series returned:

Screenshot 2024-07-10 at 11 47 02


Screenshot 2024-07-10 at 11 46 27

File: docs/sources/explore/explore-inspector.md [hunk 3]
```diff
@@ -1,97 +1,87 @@
...
+1. Select **Series joined by time** from the dropdown to view the raw data from all of your queries at once, one result set per column. You can click any column heading to sort the data.
+1. Toggle **Formatted data** to match the format in the panel.
+1. Toggle **Download for Excel** to download a CSV file specifically formatted for Excel.
+1. To download the results to a CSV file click **Download CSV** in the upper right of the Query inspector pane.
 
 ### Download log results as TXT
 
-Grafana generates a TXT file in your default browser download location. You can open it in the viewer of your choice.
+Based on the type of data source (Loki, for example), Grafana generates a TXT file of log raw data results in your default browser download location. You can open it in the viewer of your choice.
```
Comment (ifrost) [inline docs/sources/explore/explore-inspector.md:75]: We can also say that if logs are present in the result, there's an option to download logs as text.

Screenshot 2024-07-10 at 11 50 11

**Review by ifrost — APPROVED** [review]
Great stuff 🙌  I left a few minor comments but otherwise looks great!

File: docs/sources/explore/explore-inspector.md (hunk 2 above)
Comment (lwandz13) [reply docs/sources/explore/explore-inspector.md:67]: Fixed!

File: docs/sources/explore/explore-inspector.md (hunk 3 above)
Comment (lwandz13) [reply docs/sources/explore/explore-inspector.md:75]: Fixed!

**Pull request merged**